http/middleware/metrics.go

```diff
@@ -44,8 +44,9 @@ func NewMetrics(r *metrics.Registry, subsystem string, metricsAggregation bool)
 			path = c.FullPath()
 		}
 
-		requestsCount.WithLabelValues(path, fmt.Sprintf("%v", c.Writer.Status())).Inc()
-		requestsDuration.WithLabelValues(path, fmt.Sprintf("%v", c.Writer.Status())).Observe(float64(d / time.Microsecond))
+		status := strconv.Itoa(c.Writer.Status())
+		requestsCount.WithLabelValues(path, status).Inc()
+		requestsDuration.WithLabelValues(path, status).Observe(float64(d / time.Microsecond))
```

Comment:
není lepší `d.Microseconds`?

Reply:
Changed the line to call `Microseconds()` method directly as it was a quick fix. But again, refactor and polishing is somewhat out of the scope of this PR, if not otherwise suggested by the new linters.